streetview/download.py

```diff
@@ -58,17 +68,18 @@ def iter_tiles(pano_id: str) -> Generator[Tile, None, None]:
         del image
 
 
-def get_panorama(pano_id: str) -> Image.Image:
+def get_panorama(pano_id: str, zoom: int) -> Image.Image:
```

Comment:
Can we have a sensible default value for `zoom`? That way, new-comers can get up and running easily by just passing the panorama id.

Reply:
Added a default value of 5, to keep backwards compatibility with those who already used it